### ISSUE TITLE
UI Component - Editing

### ISSUE THREAD

**Issue by briansztamfater** [status-im/status-react#13726]
This issue is to track the creation of the component: Editing

You can access the link at https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=4170%3A78081

Please have a chat with the design team before starting with the component, and don't forget to add sizing to the issue

<img width="745" src="https://user-images.githubusercontent.com/18485527/181095756-1276c790-2eb8-4ee2-a1c6-a73ea06be4db.png">

**Comment by briansztamfater** [comment]
This is similar to #13538 so I will work on it

**Comment by briansztamfater** [comment]
We have some regression issues after implementing new message input and need to reimplement this, will raise a new PR with the implementation unless there's any objection cc @flexsurfer
